src/Microsoft.ML.Data/Prediction/CalibratorCatalog.cs

```diff
@@ -280,65 +263,67 @@ protected override Delegate MakeGetter(Row input, int iinfo, Func<int, bool> act
     /// The PlattCalibratorEstimator.
```

Comment:
>The PlattCalibratorEstimator. [](start = 8, length = 29)

/// <summary>
/// Creates Platt calibrator by fitting a logistic regression model to a classifier's scores
///</summary>

Reply:
Hmmm. I'll just fix the words a little, but this needs a lot more work than just this. I'm pretty sure those remarks are going to be utterly incomprehensible when written out also. I'm not quite equal to that task.